boa_interner/src/sym.rs

```diff
@@ -126,10 +130,18 @@ impl Sym {
     }
 
     /// Returns the internal value of the [`Sym`]
+    #[cfg(not(feature = "fuzzer-not-safe-for-production"))]
     #[inline]
     pub(super) const fn get(self) -> usize {
         self.value.get()
     }
+
+    /// Returns the internal value of the [`Sym`]
+    #[cfg(feature = "fuzzer-not-safe-for-production")]
+    #[inline]
+    pub const fn get(self) -> usize {
+        self.value.get()
+    }
```

Comment:
I'd just make the original `get` public.

Reply:
Completed.